security-release-process.md

```diff
@@ -225,6 +225,10 @@ These steps should be completed within the 1-7 days of Disclosure.
 
 If the CVSS score is under 4.0 ([a low severity score](https://www.first.org/cvss/specification-document#i5)) or the assessed risk is low the Fix Team can decide to slow the release process down in the face of holidays, developer bandwidth, etc. These decisions must be discussed on the security@kubernetes.io mailing list.
 
+If the CVSS score is under 7.0 (a medium severity score), the Fix Lead may choose to carry out the fix semi-publicly. This means that PRs are made directly in the public kubernetes/kubernetes repo, while restricting discussion of the security aspects to private channels. The fix lead will make the determination whether there would be user harm in handling the fix publicly that outweighs the benefits of open engagement with the community.
```

Comment:
```If the CVSS score is under 7.0 (a medium severity score), or with the agreement of the PSC, the Fix Lead...```

CVSS scoring is a very imperfect tool. We should acknowledge there's also some discretion here.

Reply:
SGTM

Reply:
Added a note.